### ISSUE TITLE
Anyone built an Australian version?

### ISSUE THREAD

**Issue by ctolkien** [Makerr-Studio/CentrifugalDustSeparator-#19]
As per title. Acrylic tubes are expensive!

**Comment by IanVicAus** [comment]
FYI the link is for 1m of 145mm ID acrylic tube (2.5mm wall thickness) @ $133 / m (enough for 2 tubes @ 500mm)
Obviously this needs a different set of sized pieces (and first the ID accurately measured)
Price wise the same as importing tube from Amazon (which is PVC not acrylic, it is also thinner ~2mm v 2.5mm).
No association tot he website
https://www.acrylicsonline.com.au/products/extruded-clear-acrylic-tube-150mm

**Comment by ctolkien** [comment]
Yep, that's the one I've been looking at - I think someone will have to pull the trigger on ordering this and figuring at the sizes.
